File: app/src/main/java/com/aavens/mindloft/db/ThingContract.java
package com.aavens.mindloft.db;

import android.provider.BaseColumns;

public class ThingContract {

    public ThingContract() {
    }

    public static abstract class ThingEntry implements BaseColumns {
        public static final String TABLE_NAME = "things";
        public static final String COLUMN_NAME_VALUE_TYPE = "type";
        public static final String COLUMN_NAME_VALUE = "value";
        public static final String COLUMN_NAME_ROOM_ID = "room_id";
        public static final String COLUMN_NAME_DATE = "date";
    }
}

File: app/src/main/java/com/aavens/mindloft/db/ThingContract.kt
package com.aavens.mindloft.db

import android.provider.BaseColumns

class ThingContract {

    abstract class ThingEntry: BaseColumns {
        companion object {
            val _ID = "_id"
            val TABLE_NAME = "things"
            val COLUMN_NAME_VALUE_TYPE = "type"
            val COLUMN_NAME_VALUE = "value"
            val COLUMN_NAME_ROOM_ID = "room_id"
            val COLUMN_NAME_DATE = "date"
        }
    }
}
